src/client/styles/styles.scss

```diff
@@ -282,3 +282,30 @@ $two-thirds: $one-third * 2;
   border-color: 	#AA2A16;
   background-color: 	#AA2A16;
 }
+
+.dashboard-table {
+  font-size: 18px;
+}
+
+.dashboard-table > .govuk-table__head .govuk-table__header {
+  vertical-align: bottom;
+  color: #1d70b8;
+}
+
+.govuk-\!-width-one-seventh {
+  width: 100% !important;
+}
+@media (min-width: 40.0625em) {
+  .govuk-\!-width-one-seventh {
+    width: 14.286% !important;
```

Comment:
is the design really 1/7th? typical design language is base 8 (or 16) or base 6 (or 12). It's because both 16 or 12 has lots of factors so easily divisible. 7 is a prime

Reply:
Good point. It's not really 1/7 it's the closest I could come to without taking too much space for the numbers, I can change it to 1/6